packages/bsp/common/etc/udev/rules.d/10-wifi-disable-powermanagement.rules

```diff
@@ -1 +1 @@
-KERNEL=="wlan*", ACTION=="add", RUN+="/sbin/iw dev %k set power_save off"
+SUBSYSTEM=="net", ACTION!="remove", ENV{DEVTYPE}=="wlan", RUN+="/usr/sbin/iw dev $name set power_save off"
```

Comment:
Is path actually needed?

Reply:
OK. got it. Other problem could be that different userland have different location - check for jammy, noble, bookworm, trixie

Reply:
Yeah, classic :)